plugins/org.locationtech.udig.libs/META-INF/MANIFEST.MF

```diff
@@ -208,7 +208,7 @@ Bundle-ClassPath: .,
  lib/gt-xsd-wfs-22.1.jar,
  lib/gt-xsd-wmts-22.1.jar,
  lib/gt-xsd-wps-22.1.jar,
- lib/guava-18.0.jar,
+ lib/guava-27.0.jar,

```

Comment:
Looks like build.properties is not up-to-date anymore. Neither jar for guava version 18.0 nore version 27.0 were added to Bundle-classpath. 

Wondering if the coverage renderer would work., see dependency-tree:

```
[INFO] +- org.geotools:gt-render:jar:22.1:compile
[INFO] |  +- org.geotools:gt-coverage:jar:22.1:compile
[INFO] |  |  +- it.geosolutions.jaiext.affine:jt-affine:jar:1.1.12:compile
[INFO] |  |  +- it.geosolutions.jaiext.algebra:jt-algebra:jar:1.1.12:compile
[INFO] |  |  +- it.geosolutions.jaiext.bandmerge:jt-bandmerge:jar:1.1.12:compile
[INFO] |  |  +- it.geosolutions.jaiext.bandselect:jt-bandselect:jar:1.1.12:compile
[INFO] |  |  +- it.geosolutions.jaiext.bandcombine:jt-bandcombine:jar:1.1.12:compile
[INFO] |  |  +- it.geosolutions.jaiext.border:jt-border:jar:1.1.12:compile
[INFO] |  |  +- it.geosolutions.jaiext.buffer:jt-buffer:jar:1.1.12:compile
[INFO] |  |  +- it.geosolutions.jaiext.crop:jt-crop:jar:1.1.12:compile
[INFO] |  |  +- it.geosolutions.jaiext.iterators:jt-iterators:jar:1.1.12:compile
[INFO] |  |  +- it.geosolutions.jaiext.lookup:jt-lookup:jar:1.1.12:compile
[INFO] |  |  +- it.geosolutions.jaiext.mosaic:jt-mosaic:jar:1.1.12:compile
[INFO] |  |  +- it.geosolutions.jaiext.nullop:jt-nullop:jar:1.1.12:compile
[INFO] |  |  +- it.geosolutions.jaiext.rescale:jt-rescale:jar:1.1.12:compile
[INFO] |  |  +- it.geosolutions.jaiext.scale:jt-scale:jar:1.1.12:compile
[INFO] |  |  +- it.geosolutions.jaiext.scale2:jt-scale2:jar:1.1.12:compile
[INFO] |  |  |  \- org.huldra.math:bigint:jar:0.7.1:compile
[INFO] |  |  +- it.geosolutions.jaiext.stats:jt-stats:jar:1.1.12:compile
[INFO] |  |  |  \- com.google.guava:guava:jar:27.0-jre:compile
```

Please check classpath-file as well

However, just a suggestion since we have an OSGi-Dependency provided by Eclipse-Opit Repository in target-definition-file:
* update pom-libs:
** add explict dependency to guava as provided in pom-libs.xml (with minimum version 27.0 **OR**
** add dependency to it.geosolutions.jaiext.stats:jt-stats and exclude com.google.guava:guava
* remove entries for guava-lib (Bundle-Classpath) in MF-File
* import com.google.common packages in MF-File

in target use from https://download.eclipse.org/tools/orbit/downloads/drops/R20201130205003/ repository 

`<unit id="com.google.guava" version="27.1.0.v20190517-1946"/>`

rather than 

`<unit id="com.google.guava" version="15.0.0.v201403281430"/>` from 
https://download.eclipse.org/tools/orbit/downloads/drops/R20180330011457/ P2 repository